pkg/controller/machineset/controller_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	machinev1 "github.com/openshift/api/machine/v1beta1"
+	"github.com/stretchr/testify/assert"
```

Comment:
Can we use gomega here instead? It's already being used within the tests in this file